app/models/ability.rb

```diff
@@ -47,6 +48,12 @@ def admin_user_permissions(_user)
     can :export, Unit
   end
 
+  def tn_administration_user_permissions(_user)
+    can :manage, Unit
```

Comment:
manage would also mean delete. sure we want that?

Reply:
Think so, @lukaseisenring ?